app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,17 @@
+class BookingsController < ApplicationController
+  def create
+    @booking = Booking.art(bookings_params)
+    @booking.user = current_user
+    if Booking.save
+      redirect_to art_path(@booking.art)
+    else
+      render 'new', status: :unprocessable_entity
+    end
+  end
```

Comment:
That's not how you create a booking.
booking = Booking.new(params)
booking.user = current_user
booking.art = Art.find(params[:art_id])

Also there are no new so you can't render it. If it's not saved you just redirect to art path too